api/config.yaml

```diff
@@ -308,7 +308,8 @@ gitleaks:
     git clone -b %GIT_BRANCH% --single-branch %GIT_REPO% code --quiet 2> /tmp/errorGitCloneGitleaks
     if [ $? -eq 0 ]; then
         touch /tmp/results.json
-        $(which gitleaks) --log=warn --report=/tmp/results.json --repo-path=./code --branch=%GIT_BRANCH% --repo-config > /tmp/errorGitleaks
+        timeout -t 300 $(which gitleaks) --log=warn --report=/tmp/results.json --repo-path=./code --branch=%GIT_BRANCH% --repo-config > /tmp/errorGitleaks ||
```

Comment:
Should we add this timeout to all other security tests? This way husky would always run in under 5 minutes.

Reply:
Absolutely, @Krlier! I totally agree. Can you create an issue so we can work on this soon, please?

Reply:
I just created this [issue](https://github.com/globocom/huskyCI/issues/452) so we can work on it later!